### DISCUSSION TITLE
请问book和incollection型的参考文献格式如何修改

### DISCUSSION THREAD

**Discussion by ginkgotyranny** [nju-lug/NJUThesis#153] [Q&A] [answered]
文章中一些参考文献是book和incollection类型：
如：
![image](https://user-images.githubusercontent.com/77227575/170696948-7bc33fbc-0b79-4829-b98a-8b8398ca7344.png)
但是编译之后，title和booktitle之间会有//分隔，如：
![image](https://user-images.githubusercontent.com/77227575/170697078-b40d1d70-5625-4b0d-b8c4-57e1a1ceed65.png)
导师让把//去掉，想请问一下如何修改。

**Comment by note286** [comment, marked as answer]
[biblatex-gb7714-2015.pdf](http://mirrors.ctan.org/macros/latex/contrib/biblatex-contrib/biblatex-gb7714-2015/biblatex-gb7714-2015.pdf)中查看`gbpunctin`选项。
